libgraph/include/katana/GraphTopology.h

```diff
@@ -1608,10 +1730,20 @@ class KATANA_EXPORT PGViewCache {
       PropertyGraph* pg, const RDGTopology::TransposeKind& tpose_kind,
       const RDGTopology::EdgeSortKind& sort_kind, bool pop) noexcept;
 
+  std::shared_ptr<EdgeShuffleTopology> BuildOrGetEdgeShuffTopoImpl(
+      PropertyGraph* pg, const RDGTopology::TransposeKind& tpose_kind,
+      const RDGTopology::EdgeSortKind& sort_kind, bool pop,
+      const std::string& property_name) noexcept;
+
   std::shared_ptr<EdgeShuffleTopology> BuildOrGetEdgeShuffTopo(
       PropertyGraph* pg, const RDGTopology::TransposeKind& tpose_kind,
       const RDGTopology::EdgeSortKind& sort_kind) noexcept;
 
+  std::shared_ptr<EdgeShuffleTopology> BuildOrGetEdgeShuffTopo(
```

Comment:
Drop the enum parameter and rename to convey that it's for building edge sorted by property.